de.metas.business.rest-api-impl/src/main/java/de/metas/rest_api/invoice/impl/InvoicePDFService.java

```diff
@@ -0,0 +1,79 @@
+package de.metas.rest_api.invoice.impl;
+
+import java.util.List;
+
+import org.adempiere.archive.api.IArchiveBL;
+import org.adempiere.archive.api.IArchiveDAO;
+import org.adempiere.invoice.service.IInvoiceDAO;
+import org.adempiere.util.lang.impl.TableRecordReference;
+import org.compiere.model.I_AD_Archive;
+import org.compiere.model.I_C_Invoice;
+import org.compiere.util.Env;
+import org.springframework.stereotype.Service;
+
+import com.google.common.base.Optional;
```

Comment:
Now that we finally have it in the standard-JAva, please use `java.util.Optional` instead of guava's variant

Reply:
done